.github/workflows/package.yaml

```diff
@@ -0,0 +1,13 @@
+name: Package
+
+on:
+  push:
+    branches:
+      - 'pkg/*'
+
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: full
+
+jobs:
```

Comment:
Jobs is empty?

Reply:
It was a mistake.
The unready `workflow` was removed.